Columbus/frontend/src/components/Notifications/NotificationMenu.js

```diff
@@ -0,0 +1,83 @@
+import {React, useState, useEffect} from "react";
+
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import Divider from '@mui/material/Divider';
+            import ListItem from '@mui/material/ListItem';
+
+import Notification from "./Notification";
+import USER_SERVICE from '../../services/user';
+
+export default function NotificationMenu(props) { 
+
+const { isNotificationsOpen, setIsNotificationsOpen, isNotificationMenuOpen } = props;
+
+const [notifications, setNotifications] = useState([]);
+const [isLoading, setIsLoading] = useState(true);
+
+useEffect(() => {
+
+        USER_SERVICE.GET_NOTIFICATIONS(localStorage.getItem('username'))
+            .then((res) => {
+              setNotifications(res.data.notifications.orderedItems)
```

Comment:
Sanırım kullanıcının notificationı olmayınca response aşağıdaki gibi oluyor. Bu yüzden de eğer kullanıcıya notification gelmemişse burası patlayabilir :Fyi

<img width="958" alt="Screen Shot 2022-01-01 at 18 07 32" src="https://user-images.githubusercontent.com/34331026/147853616-ac2df6f8-1922-4826-af2b-d548f0d1bc5c.png">

Reply:
Belki burada `res.data?.notifications` işe yarayabilir ama emin değilim. Diğer türlü notifications objesi olmadığı için crash olması yüksek olası

Reply:
Yes that is true, I will look at it again

Reply:
I wrote this according to the previous endpoint, now I have updated `notifications` to `other_notifications`. It seems to work in this case.